README.md

```diff
@@ -1,36 +1,533 @@
+
+```bash
+npm install --force
+```
+
+## API documentation
+
+<!--eng: Relevant for this repo? Both links are 404s right now. Will these doc links eventually provide access to SDK docs generated from code annotations added to this repo?-->
 
 - [Documentation for the main branch](https://github.com/expo/expo/blob/main/docs/pages/versions/unversioned/sdk/xmtp-react-native-sdk.md)
+
 - [Documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/xmtp-react-native-sdk/)
 
-# Installation in managed Expo projects
+<!--eng: I basically created two paths for install - one for a managed Expo project and one for a bare RN project. The dev takes one route or the other depending on their setup, is that correct?-->
+
+## Install in a managed Expo project
+
+<!--eng: Is this correct? There is no API documentation for this right now - so is install for an expo project not possible yet? Or do we have a command we want to share here?-->
 
-For [managed](https://docs.expo.dev/versions/latest/introduction/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](#api-documentation). If you follow the link and there is no documentation available then this library is not yet usable within managed projects &mdash; it is likely to be included in an upcoming Expo SDK release.
+For a [managed](https://docs.expo.dev/versions/latest/introduction/managed-vs-bare/) Expo project, follow the installation instructions in the [API documentation](#api-documentation) for the latest stable release. If you follow the link and there is no documentation available, then this library is not yet usable within managed projects. It is likely to be included in an upcoming Expo SDK release.
 
-# Installation in bare React Native projects
+## Install and configure in a bare React Native project
+
+<!--eng: What does this mean - why do I need to install the Expo package if this is a bare RN project? Is there a command we want to share here?-->
 
 For bare React Native projects, you must ensure that you have [installed and configured the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/) before continuing.
 
 ### Add the package to your npm dependencies
 
-```
+```bash
 npm install xmtp-react-native-sdk
 ```
 
 ### Configure for iOS
 
-Run `npx pod-install` after installing the npm package.
+<!--eng: For this command and the one below - what are we configuring for iOS and for Android? When does this configuration happen? Do I run these commands in the root dir?-->
+
+After installing the npm package, run:
+
+```bash
+npx pod-install
+```
 
 ### Configure for Android
 
-# Contributing
+After installing the npm package, run:
+
+<!--eng: Just guessing at this based on talking with ChatGPT LOL. Says that starting from React Native 0.60, this command may not be necessary.-->
+
+```bash
+react-native link xmtp-react-native-sdk
+```
+
+## Usage overview
```

Comment:
I think while stuff is still in this super early phase, since a lot of the APIs from xmtp-js are going to be missing, I'd say maybe we just point people to the [example app](https://github.com/xmtp/xmtp-react-native/tree/main/example/src) for usage. Once we've got stuff more fleshed out, we can add this section (which should basically just mirror the usage from xmtp/xmtp-js).